docs/debug-guide.md

```diff
@@ -84,6 +86,10 @@ about `go tool pprof`. My go-to method of analyzing these is to run the `web`
 command, which generates an SVG dotgraph and opens it in your browser. This is
 the quickest way to easily point out where the hot spots in the code are.
 
+### Analysing vars and memory statistics
```

Comment:
nit: could you also add an entry in the "Table of Contents" above pointing here?

Reply:
Drat, my bad, will do